config/sync/context.context.openseadragon_block.yml

```diff
@@ -7,54 +7,54 @@ dependencies:
     - openseadragon
 _core:
   default_config_hash: sN9l77XqTzoy1x5fp1O7v91Di5_6iCx9prNHDvGYrKo
-name: openseadragon_block
 label: 'Paged Content - Openseadragon'
+name: openseadragon_block
 group: Display
 description: 'Display a block for Paged Content'
 requireAllConditions: false
 disabled: false
 conditions:
   node_has_term:
     id: node_has_term
-    negate: 0
+    negate: false
     uuid: e5689bd5-7eec-4378-b329-2f35d5bb35b0
-    uri: 'http://openseadragon.github.io,https://schema.org/Book'
     context_mapping:
       node: '@node.node_route_context:node'
+    uri: 'http://openseadragon.github.io,https://schema.org/Book'
     logic: and
 reactions:
   blocks:
+    id: blocks
+    uuid: 50b9b25e-0836-4531-8a78-9698d71d81de
     blocks:
       4fe759c9-e1bf-4709-871c-7b8a407c840b:
+        uuid: 4fe759c9-e1bf-4709-871c-7b8a407c840b
         id: openseadragon_block
         label: 'Openseadragon block'
-        label_display: 0
         provider: openseadragon
+        label_display: '0'
```

Comment:
I don't know where you can see these schemas (or if you can) but I noticed this `label_display` has a value of `'0'` (a string) and below (line 45 -> 49) it is `label_display: visible`. So does `'0'` mean "visible" or "hidden" or something else?

Reply:
Strictly, this should be referring to https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/config/schema/core.data_types.schema.yml#L303-305 (via https://github.com/Islandora/openseadragon/blob/8f9eeff9d3be213dfa5e0477959df5882776cb44/config/schema/openseadragon.schema.yml#L15-L16)... we're dealing with the settings of a block here which should be defined as `block_settings`; however, I seem to recall digging in to things a bit in the semi-recent past, and the placing of blocks via contexts breaks the associations back to the underlying schema definitions... instead [rolling their own schema which seems like copypasta of the core `block_settings` type](https://git.drupalcode.org/project/context/-/blob/8.x-4.x/config/schema/context.data_types.schema.yml#L18-52) with some additional context specific properties.

Reply:
Digging deeper, the concrete value seems to come from the block plugin; for example, [the trait associated with the `BlockPluginInterface` defines a checkbox for this property](https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/lib/Drupal/Core/Block/BlockPluginTrait.php#L180-185)... When checked, would give the `#return_value` of [`BlockPluginInterface::BLOCK_LABEL_VISIBLE`](https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/lib/Drupal/Core/Block/BlockPluginInterface.php#L29)... when unchecked, 0.

Reply:
So maybe some of these are just stale? I notice both this-here-referenced block config with `label_display: 0` and the one below (lines 45-49 with `label_display: visible`) are both for the `bartik` theme. We haven't used that in ages! And we have documented the need to [update all of them to use Olivero](https://github.com/Islandora/islandora-starter-site/issues/10). Hopefully that will flush out any stale config language in these contexts.